client/src/ui/components/SettingsDropDown.tsx

```
@@ -34,9 +35,11 @@ export const SettingsDropDown = () => {
           Account
         </DropdownMenuLabel>
         {ACCOUNT_CONNECTOR === "controller" && (
-          <div className="p-1 flex flex-col gap-2">
-            <AccountDetails />
-          </div>
+          <>
+            <div className="p-1 flex flex-col gap-2">
+              <AccountDetails />
+            </div>
+          </>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)